drivers/dma/Kconfig.dma_pl330

```diff
@@ -0,0 +1,9 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+config	DMA_PL330
```

Comment:
nit: delete the extra space between config and DMA_PL330

Reply:
ack